apps/web/src/presentation/components/launchpad/abc/ABCSettingsModelController.ts

```diff
@@ -3,18 +3,26 @@ import { DAOCreationRepository } from "../../../../domain/repository/DAOCreation
 import { setABCConfig } from "../../../../domain/use-case/DAOCreationUseCases";
 import { ABCConfig } from "../../../../domain/model/ABCConfig";
 import { TokenInfo } from "../../../../domain/model/TokenInfo";
+import { fetchBalance, getAccount } from "@wagmi/core";
+import { parseEther } from "viem";
 
 export function useABCSettingsModelController(daoCreationRepository: DAOCreationRepository) {
     const [augmentedBondingCurveSettings, setAugmentedBondingCurveSettings] = useState<ABCConfig>(new ABCConfig(
         daoCreationRepository.getDAOInfo().getABCConfig().reserveRatio ?? 0,
         daoCreationRepository.getDAOInfo().getABCConfig().getReserveInitialBalance() ?? 0,
         daoCreationRepository.getDAOInfo().getABCConfig().entryTribute ?? 0,
         daoCreationRepository.getDAOInfo().getABCConfig().exitTribute ?? 0,
-        new TokenInfo(daoCreationRepository.getDAOInfo().getABCConfig().collateralToken?.tokenAddress ?? '', daoCreationRepository.getDAOInfo().getABCConfig().collateralToken?.tokenSymbol ?? ''),
+        new TokenInfo(
```

Comment:
The constructor we are using is different, but I don't see any changes on TokenInfo file/class. Is it because it was not working before?

Reply:
It was not working before since we changed the TokenInfo.ts to be able to retrieve available tokens from data. Current constructor is as follows:

`    constructor(tokenName?: string, tokenSymbol?: string, tokenAddress?: string, tokenLogo?: string) {
        this.tokenName = tokenName;
        this.tokenSymbol = tokenSymbol;
        this.tokenAddress = tokenAddress;
        this.tokenLogo = tokenLogo;
    }`